modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/accept/language/AcceptLanguageImpl.java

```diff
@@ -82,6 +84,13 @@ else if (acceptLanguage.equals("zh-Hant-TW")) {
 				});
 
 			if (ListUtil.isEmpty(locales)) {
+				if (Objects.equals(
```

Comment:
The if clause could be simplified. Please, look for the same pattern in the portal:

![image](https://github.com/liferay-headless/liferay-portal/assets/32699538/fec314f3-3725-4703-ad36-dd520e4b3f18)